src/config.rs

```diff
@@ -40,6 +40,25 @@ impl Default for TeleopConfig {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct RosoutConfig {
+    pub buffer_size: usize,
+    pub min_loglevel: i8,
+    pub screen_percentage: u16,
+    pub enabled_by_default: bool,
+}
+
+impl Default for RosoutConfig {
+    fn default() -> RosoutConfig {
+        RosoutConfig {
+            buffer_size: 150,
+            min_loglevel: 2, // INFO, see rosmsg show rosgraph_msgs/Log
+            screen_percentage: 25,
+            enabled_by_default: true,
```

Comment:
I would disable it by default, that will allow a quick glance at the robot on the complete terminal. Enabling it on demand with l is quick enough. But that might just be me. Any thumb up thumb downs for this?